org.gnome.SoundJuicer.json

```diff
@@ -112,8 +124,8 @@
                     "type": "git",
                     "disable-submodules": true,
                     "url": "https://gitlab.freedesktop.org/gstreamer/gstreamer.git",
-                    "commit" : "8dbfc89a850d484a27937eb882978251bfce06b3",
-                    "tag": "1.20.2"
+                    "commit" : "da69285863780ce0ebb51482edcf1d54c7c29533",
+                    "tag": "1.24.3"
```

Comment:
This wasn't tested I think.

We only build plugins, so the version of GStreamer used needs to be the same one as in the runtime. It uses `1.22.11`, `1.24.3` is too new.

Reply:
I've actually tested it by successfully ripping two CDs to MP3. But that's good to know, I'll revert gstreamer to 1.22.11 for this pull request.

How do I find out which versions of which packages are present in which runtimes, for future reference? I can't seem to find out how org.gnome.platform is built.

Reply:
I got those warnings, and assumed it wouldn't work, but you're right, it still should:
```shell
$ flatpak run org.gnome.SoundJuicer//test

(sound-juicer:2): GStreamer-WARNING **: 11:16:59.135: External plugin loader failed. This most likely means that the plugin loader helper binary was not found or could not be run. You might need to set the GST_PLUGIN_SCANNER environment variable if your setup is unusual. This should normally not be required though.

(sound-juicer:2): GStreamer-WARNING **: 11:16:59.301: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstnvcodec.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstnvcodec.so: undefined symbol: gst_av1_picture_get_user_data

(sound-juicer:2): GStreamer-WARNING **: 11:16:59.811: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstva.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstva.so: undefined symbol: gst_mpeg2_picture_set_user_data
```

I used `gst-inspect` inside the container to check the version, although you could also go read `/usr/manifest.json` there, which will tell you the exact version.